.github/workflows/release.yml

```
@@ -0,0 +1,148 @@
+        uses: mParticle/mparticle-workflows/.github/workflows/sdk-release-repo-branch-check.yml@stable
+
+    build-and-test:
+        name: Build and Test
+        runs-on: ubuntu-latest
+        needs: confirm-public-repo-master-branch
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: NPM install
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 18.x
+
+            - name: Run NPM CI
+              run: npm ci
+
+            - name: Build Files
+              run: npm run build
+
+            - name: Run Core tests
+              run: npm run test
+
+            - name: Archive npm failure logs
+              uses: actions/upload-artifact@v3
+              if: failure()
+              with:
+                  name: npm-logs
+                  path: ~/.npm/_logs
+
+    create-release-branch:
+        name: Create release branch
+        runs-on: ubuntu-latest
+        needs:
+            - build-and-test
+            - confirm-public-repo-master-branch
+        steps:
+            - name: Checkout development branch
+              uses: actions/checkout@v3
+              with:
+                  repository: mparticle-integrations/mparticle-javascript-integration-twitter
+                  ref: development
+
+            - name: Create and push release branch
+              run: |
+                  git checkout -b release/${{ github.run_number }}
+                  git push origin release/${{ github.run_number }}
+    release:
+        name: Perform Release
+        runs-on: ubuntu-latest
+        needs:
+            - build-and-test
+            - create-release-branch
+            - confirm-public-repo-master-branch
+        env:
+            GITHUB_TOKEN: ${{ secrets.MP_INTEGRATIONS_SEMANTIC_RELEASE_BOT }}
+            GIT_AUTHOR_NAME: mparticle-automation
+            GIT_AUTHOR_EMAIL: developers@mparticle.com
+            GIT_COMMITTER_NAME: mparticle-automation
+            GIT_COMMITTER_EMAIL: developers@mparticle.com
+            NPM_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+
+        steps:
+            - name: Checkout public master branch
+              uses: actions/checkout@v3
+              with:
+                  fetch-depth: 0
+                  ref: master
+
+            - name: Import GPG Key
+              uses: crazy-max/ghaction-import-gpg@v4
+              with:
+                  gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+                  passphrase: ${{ secrets.GPG_PASSPHRASE }}
+                  git_user_signingkey: true
+                  git_commit_gpgsign: true
+
+            - name: Merge release branch into master branch
+              run: |
+                  git pull origin release/${{ github.run_number }}
+            - name: Setup Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16.x
```

Probably should update this to something like v18 or v20

```suggestion
                  node-version: 20.x
```